src/sonic-yang-models/yang-models/sonic-system-defaults.yang

```diff
@@ -0,0 +1,41 @@
+module sonic-system-defaults{
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/system-defaults";
+    prefix system-defaults;
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    description "SYSTEM_DEFAULTS Table yang Module for SONiC";
+
+    container sonic-system-defaults {
+
+        container SYSTEM_DEFAULTS {
+
+            description "system_defaults table in config_db.json";
+
+            list SYSTEM_DEFAULTS_LIST {
+
+                key "name";
+
+                leaf name {
+                    description "feature name in SYSTEM_DEFAULTS table";
+                    type string {
+                        length 1..32;
+                    }
+                }
+
+                leaf status {
+                    description "default status of the feature";
+                    type string {
+                        pattern "enabled|disabled";
+                    }
```

Comment:
Can you use stypes:admin_status?

Reply:
I think the value of `stypes:admin_status` is `up` or `down`. I think it would be better to use `enabled` or `disabled` here.

Reply:
https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/yang-templates/sonic-types.yang.j2#L126-L131
How about admin_mode?

Reply:
Yes, `admin_mode` would be good. Updated.